packages/check/match.js

```diff
@@ -145,33 +139,41 @@ var typeofChecks = [
   [undefined, "undefined"]
 ];
 
-var checkSubtree = function (value, pattern) {
+// Return `false` if it matches. Otherwise, return an object with a `message` and a `path` field.
+var testSubtree = function (value, pattern) {
```

Comment:
This doesn't correspond to `test` the way `checkSubtree` corresponded to `check`.